src/core/aws-client-http_utils.adb

```diff
@@ -336,12 +336,23 @@ package body AWS.Client.HTTP_Utils is
       Parse_Header (Connection, Result, Keep_Alive);
 
       declare
-         TE     : constant String  :=
```

Comment:
I would avoid changing that, this is just add use clause which does not make the code clearer to me.

Reply:
use removed, package prefixes restored